src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -44,7 +52,7 @@ const styles = StyleSheet.create({
     width: 24,
     height: 24,
     borderRadius: 12,
-    marginRight: 4
+    marginRight: AVATAR_MARGIN
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The margin right value has been replaced by a constant `AVATAR_MARGIN`. This is good for maintainability as it allows you to change the margin from a single place if needed in the future.
```diff
-    marginRight: 4
+    marginRight: AVATAR_MARGIN
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->